libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1785,7 +1785,7 @@ void IRGeneratorForStatements::endVisit(MemberAccess const& _memberAccess)
 					""
 				);
 				define(IRVariable{_memberAccess}) << formatNumber(
-					util::selectorFromSignature(functionType.externalSignature())
+					util::selectorFromSignatureU256(functionType.externalSignature())
```

Comment:
We use a large variety of these helpers which could be further refactored/simplified. For example I doubt we need `formatNumber` here and instead should use `selectorFromSignatureH32().hex()`.